dataflow/gpu-examples/tensorflow-landsat/Dockerfile

```diff
@@ -37,5 +37,11 @@ RUN apt-get update \
     && pip check
 
 # Set the entrypoint to Apache Beam SDK worker launcher.
-COPY --from=apache/beam_python3.8_sdk:2.34.0 /opt/apache/beam /opt/apache/beam
-ENTRYPOINT [ "/opt/apache/beam/boot" ]
+# COPY --from=apache/beam_python3.8_sdk:2.34.0 /opt/apache/beam /opt/apache/beam
+# ENTRYPOINT [ "/opt/apache/beam/boot" ]
+
+# Workaround for b/199755022
```

Comment:
We might want to wait until this fix is published before we merge, probably by Beam 2.35 (?)

Reply:
This works with Beam 2.35 RC3